app/util.js

```diff
@@ -109,12 +109,19 @@ const searchword = location => {
             const q = location.replace(`${word} `, "")
             if (q && location.startsWith(`${word} `)) {
                 const queries = q.split(",").map(item => item.trim())
```

Comment:
I don't think we specifically need a trim here actually, as the remainder might have spaces and commas mixed that users would expect to keep and be inserted in the final search part. Maybe add a note to the help about how the remainder works and that spaces are kept.